test/test_feed.cpp

```diff
@@ -19,30 +18,27 @@ using std::string;
 etree::Element atomFeed("x");
 etree::Element rss20Feed("x");
 
-MU_SETUP(loadFeeds)
+TEST_CASE("loadFeeds", "[feed]")
```

Comment:
Aah! I completely misunderstood TEST_CASE vs. SECTION. It seems this will behave exactly how things used to, except now the global variables are gone.. fabulous! :-)